NEWS.md

```diff
@@ -1,5 +1,6 @@
 MixedModels v4.10.0 Release Notes
 ==============================
+* Rank deficiency in prediction is now supported, both when the original model was fit to rank-deficient data and when the new data are rank deficient. The behavior is consistent but potentially be surprising when both old and new data are rank deficient. See the `predict` docstring for an example. [#676]
```

Comment:
"but may be surprising ..." instead of "but potentially be surprising ..."?

Reply:
```suggestion
* Rank deficiency in prediction is now supported, both when the original model was fit to rank-deficient data and when the new data are rank deficient. The behavior is consistent but may be surprising when both old and new data are rank deficient. See the `predict` docstring for an example. [#676]
```